src/app/(rucio)/layout.tsx

```diff
@@ -14,12 +16,33 @@ export default function RootLayout({
 }: {
     children: React.ReactNode
 }) {
-
+    const [siteHeaderViewModel, setSiteHeaderViewModel] = useState<SiteHeaderViewModel>({
+        status: "error",
+        homeUrl: ""
+    })
+    const fetchAccounts = () => {
```

Comment:
we should wrap fetch into a [useQuery](https://tanstack.com/query/v4/docs/framework/react/reference/useQuery)